plugins/auth-keycloak/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@deephaven/js-plugin-auth-keycloak",
+  "version": "0.2.0",
+  "description": "Deephaven Keycloak authentication plugin",
+  "keywords": [
+    "Deephaven",
+    "plugin",
+    "deephaven-js-plugin",
+    "auth",
+    "authentication",
+    "keycloak"
+  ],
+  "author": "Deephaven Data Labs LLC",
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/deephaven/deephaven-js-plugins"
+  },
+  "bugs": {
+    "url": "https://github.com/deephaven/deephaven-js-plugins/issues"
+  },
+  "homepage": "https://github.com/deephaven/deephaven-js-plugins",
+  "scripts": {
+    "start": "vite build --watch",
+    "build": "vite build"
+  },
+  "devDependencies": {
+    "@types/plotly.js": "^2.12.18",
+    "@types/react": "^17.0.2",
+    "@vitejs/plugin-react-swc": "^3.0.0",
+    "react": "^17.0.2",
+    "typescript": "^4.5.4",
+    "vite": "~4.1.4"
+  },
+  "peerDependencies": {
+    "react": "^17.0.2"
+  },
+  "dependencies": {
+    "@deephaven/auth-plugins": "0.33.1-auth-plugins.25",
+    "@deephaven/components": "0.33.1-auth-plugins.25",
+    "@deephaven/jsapi-types": "0.33.1-auth-plugins.25",
+    "@deephaven/log": "0.33.1-auth-plugins.25",
+    "@deephaven/utils": "0.33.1-auth-plugins.25",
```

Comment:
Guessing these will be updated when we merge logout and publish a new version?

Reply:
Yes, will merge after logout has gone in.